### PR TITLE
Defaults before user config for Bars module

### PR DESCRIPTION
**Pull request by Aaahh**
Not the cleanest, but in theory bars-user.glsl could be the same file as bars.glsl, so we only need to make a symlink at bars.glsl to the /etc/bars.glsl and copy bars.glsl to bars-user.glsl (so we don't really need the bars-user.glsl file in the dir, we could just copy the same file as bars-user.glsl and the file in the /etc to be symlinked at bars.glsl)

### CODE REVIEW

**Comment by Aaahh** [comment]
fixes issues mentioned in https://github.com/wacossusca34/glava/pull/73

File: shaders/bars-user.glsl [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#define DIRECTION 0
+/* Whether to switch left/right audio buffers */
+#define INVERT 0
+/* Whether to flip the output vertically */
+#define FLIP 0
+/* Whether to mirror output along `Y = X`, causing output to render on the left side of the window */
+/* Use with `FLIP 1` to render on the right side */
+#define MIRROR_YX 0
+
+#include ":bars-user.glsl"
```
Comment (Aaahh) [inline shaders/bars-user.glsl]: mistake

File: shaders/bars.glsl [hunk 2]
```diff
@@ -1,32 +1,74 @@
...
+
 /* Bar color */
-#define COLOR (#3366b2 * GRADIENT * ALPHA)
+#ifndef COLOR
+    #define COLOR (#3366b2 * GRADIENT * ALPHA)
+#endif
+
 /* Direction that the bars are facing, 0 for inward, 1 for outward */
-#define DIRECTION 0
+#ifndef BAR_WIDTH)
```
Comment (Aaahh) [inline shaders/bars.glsl]: mistake 2

**Comment by Aaahh** [comment]
Sorry for the spam, I'll finish the rest up in a few hours along with glava.c if u like the solution

**Comment by wacossusca34** [comment]
Although this _works_, it's the kind of repetition I would prefer to avoid by implementing the solution I described in #74. Having repetition with regards to user defines also makes adding new options less intuitive.

Also, having symlinked extra files for defaults in the glava configuration root will probably cause users erroneously trying to edit the defaults. Either glava should read directly from `/etc/xdg/glava` to obtain these, or defaults should be symlinked into a `~/.config/glava/defaults` directory.

**Pull request closed without merging**